capirca/lib/juniper.py

```diff
@@ -655,6 +662,8 @@ def __str__(self):
           config.Append('next-ip6 %s;' % str(self.term.next_ip[0]))
       if self.term.encapsulate:
         config.Append('encapsulate %s;' % str(self.term.encapsulate))
+      if self.term.decapsulate:
```

Comment:
We probably want this in an elif block to ensure only one or the other is used. 

"encapsulate" and "decapsulate" are terminating actions. In a given firewall filter term, we cannot have 2 different terminating actions.

So we'd also want to error out somewhere if both encapsulate and decapsulate terminating actions are encountered (and an associated test for this). 

Thanks!

Reply:
Shouldn't this be caught by the following? Happy to move to `elif` if you prefer :-)
```python
 568     if self.term.encapsulate:
 569       unique_actions.add('encapsulate')
 570     if self.term.decapsulate:
 571       unique_actions.add('decapsulate')
```

Reply:
Had another look at this and added and explicit test for `ecanpsulate`+ `decapsulate` together, which already was caught by the check above :-)